javascript/javascript1/week3/smart_phone.js

```diff
@@ -0,0 +1,48 @@
+const activities=[];
+//Adding activities
+function addActivity(date, activity, duration){
+    if(typeof(date)==="string" && typeof(activity)==="string" && typeof(duration)==="number"){
+        const newActivity={};
```

Comment:
`const newActivity={date:date, activity: activity, duration: duration}` will also work. To avoid confusion, you can rename the function arguments, then it will be: `function addActivity(date_in, activity_in, duration_in)` and 
`const newActivity={date:date_in, activity: activity_in, duration: duration_in}`